### PR TITLE
docs(INTRO): update current state section

### PR DESCRIPTION
**Pull request by divyanshu013**
Updates the current state section
- Removes image showing current state (hard to maintain)

### CODE REVIEW

**Review by kamleshchandnani — APPROVED** [review]

File: packages/blade/docs/guides/Intro.stories.mdx [hunk 1]
```diff
@@ -61,5 +61,4 @@ This package is under **maintenance** and it won't have any major releases. It w
 
 # ⌛️ Current State
 
-Here's a glimpse of where we are currently in our journey of building the design system.
-![phases-of-blade](https://user-images.githubusercontent.com/11384858/186387746-6233169f-3b1b-4d81-9f6e-7308dea7b273.png)
+For the current state of things please refer to the left hand sidebar. We're actively working on adding more components, utilities and docs. Please reach us on Slack at `#platform_fe_core` for any queries or create an issue on the GitHub repo.
```
Comment (anuraghazra) [inline packages/blade/docs/guides/Intro.stories.mdx:64]: IDEA: For current state we can create a storybook recipes in a grid with all the components.

Comment (chaitanyadeorukhkar) [inline packages/blade/docs/guides/Intro.stories.mdx:64]: Lets encourage the usage of the dedicated channel instead?
```suggestion
For the current state of things please refer to the left hand sidebar. We're actively working on adding more components, utilities and docs. Please reach us on Slack at `#design-system` for any queries or create an issue on the GitHub repo.
```

Comment (divyanshu013) [reply packages/blade/docs/guides/Intro.stories.mdx:64]: Will increase the scope 😬

Comment (anuraghazra) [reply packages/blade/docs/guides/Intro.stories.mdx]: Yeah we can keep it in backlog. Anyway we need a kitchen sink kind of story. not a huge effort :p

Comment (divyanshu013) [reply packages/blade/docs/guides/Intro.stories.mdx]: :godmode:

**Review by chaitanyadeorukhkar — APPROVED** [review]

**Pull request merged**